BlueShift-iOS-SDK/BlueShiftDeviceData.m

```diff
@@ -78,7 +78,7 @@ - (NSString *)deviceType {
     return [[UIDevice currentDevice] model];
 }
 
-- (NSString *)networkCarrierName {
+- (NSString *)getNetworkCarrier {
```

Comment:
I'd suggest renaming this to `getNetworkCarrierName`. Will be more readable in the below code.